task.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListner
+const number1 = Number(prompt('First number'));
+const number2 = Number(prompt('Second number'));
+
+const answerplus = (number1 + number2)
```

Comment:
brackets is redundant
```suggestion
const answerplus = number1 + number2
```